heron/packing/src/java/com/twitter/heron/packing/binpacking/FirstFitDecreasingPacking.java

```diff
@@ -204,6 +238,17 @@ public void close() {
   }
 
   /**
+   * Get the instances' allocation based on the First Fit Decreasing algorithm
+   *
+   * @return Map &lt; containerId, list of InstanceId belonging to this container &gt;
+   */
+  private Map<Integer, List<String>> getFFDAllocation(PackingPlan currentPackingPlan,
+                                                      Map<String, Integer> componentChanges) {
+    int numContainers = currentPackingPlan.getContainers().size();
+    return placeInstances(componentChanges, numContainers);
```

Comment:
If I understand correctly, `placeInstances` adds new containers beyond `numContainers`. Is that correct? If so can we make the signature and javadocs for `placeInstances` reflect that, because currently the method description and `numContainers` field imply that the method places instances within `numContainers` not beyond.